src/hexo/themes/sonarwhal/source/images/back-to-top.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.1 15">
+  <path fill="#fff" d="M12.9 0l13.2 13.2-1.8 1.8L12.9 3.5 1.8 15 0 13.2 12.9 0z"/>
+</svg>
```

Comment:
This can be smaller:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.1 15"><path fill="#fff" d="M12.9 0l13.2 13.2-1.8 1.8L12.9 3.5 1.8 15 0 13.2 12.9 0z"/></svg>
```

Reply:
FYI: [Made this modification](https://github.com/sonarwhal/sonarwhal.com/commit/ca5e87af67c4af7f45f1ed560f23c177b329723d#diff-8ac4d7743db650ceadb0a132ff66e2ba) before merging.